src/private/interfaces.ts

```diff
@@ -120,3 +120,77 @@ export interface UserJoinedTeamsInformation {
    */
   userJoinedTeams: TeamInformation[];
 }
+
+/**
+ * @private
+ * Hide from docs
+ *
+ * Data structure to represent a meeting room.
+ */
+export interface MeetingRoomInfo {
+  /**
+   * endpoint id of the meeting room.
+   */
+  endpointId: string;
+  /**
+   * device name of the meeting room.
+   */
+  deviceName: string;
+  /**
+   * client type of the meeting room.
+   */
+  clientType: string;
+  /**
+   * client version of the meeting room.
+   */
+  clientVersion: string;
+}
+/**
+ * @private
+ * Hide from docs
+ *
+ * Data structure to represent capabilities of a meeting room.
+ */
+export interface MeetingRoomCapability {
+  /**
+   * media control capabilities, value can be "toggleMute", "toggleCamera", "toggleCaptions", "volume".
```

Comment:
NIT: Capitalize comments for consistency with existing code

Reply:
Sure. Thanks.